x-pack/filebeat/input/httpjson/input.go

```diff
@@ -159,6 +161,20 @@ func run(
 	return nil
 }
 
+/*
+The Request.Tracer.Filename may have ":" when a httpjson input has cursor config
+The MacOs Finder will treat this as path-separator and causes to show up strange filepaths.
+This function will sanitize characters like ":" and "/" to replace them with "_" just to be
+safe on all operating systems.
+*/
+func sanitizeFilePaths(ctx v2.Context) string {
```

Comment:
```suggestion
func sanitizeFileName(name string) string {
```
and work and return on name.
(not a full path and the use of v2.Context as the parameter type is a little confusing)

Do you mind using `//` comments?

Reply:
New to Golang.. Thought a multiline comment takes `/* */`.. But  gladly take in your suggestions

Reply:
They're both valid, but `//` is more common in this context.